TheOracle2/Interactions/SlashCommands/AskTheOracleCommand.cs

```diff
@@ -12,17 +12,17 @@ public OracleAskPaths(Random random)
         this.random = random;
     }
 
-    [SlashCommand("ask", "Ask the oracle based on the predefined likelihoods")]
+    [SlashCommand("ask", "Ask the Oracle a yes/no question (p. 225). To roll on a specific oracle table, use /oracle.")]
     public async Task AskTheOracle(
-        [Summary(description: "The odds of receiving a 'yes'.")]
```

Comment:
this one's a legit bugfix 🤦  currently they're inverted from where they ought to be